docs/pages/blog/material-ui-is-now-mui.md

```diff
@@ -0,0 +1,99 @@
+
+It's the context in which we started working on v5 in 2019.
+Our primary focus was to revamp the **customization DX**.
+It had become clear that design and DX were key to unlock the next stage of growth.
+
+The release of this rebranding matches with the release of v5.0.0.
+It's a big deal.
+The new branding aligns with the increase of the company's area of focus.
+
+Our ultimate goal is to become the most effective and efficient tool to build UIs while making it accessible to the many.
+
+## Our new brand
+
+Let's dig in.
+
+### A new name
+
+We are breaking the strong association with Material Design.
+We have seen too many people confusing Material-UI with a Google effort or a synonym of Material Design.
+
+We are now called **MUI**, it stands for **M**aterials to Build **UI**s.
+and is pronounced [/ɛmjuːaɪ/](http://ipa-reader.xyz/?text=%C9%9Bmju%CB%90a%C9%AA).
+
+It's shorter, it distinguishes ourselves, it's already how many people abbreviate us.
+More importantly, it prepares the release of unstyled/headless components, a second design system to complement Material Design, and more.
+
+### A new domain name
+
+A new name wouldn't resonate without a new domain name.
+We have moved home from https://material-ui.com/ to https://mui.com/, an 3 domain laters domain.
+
+### New logos
+
+The logo of the company is evolving:
+
+- We have reduced the emphasis on the smart 3D perspective. It's a step away from the notion of elevation in Material Design with a single color.
+- We have rebalanced the proportions and equilibrium of the letters.
+- We have added a slight border-radius to soften the angles.
+
+<img loading="lazy" src="/static/blog/material-ui-is-now-mui/old-new.png" alt="" style="width: 302px; margin-bottom: 16px;" />
+
+We are introducing logos for each of our products:
+
+<img loading="lazy" src="/static/blog/material-ui-is-now-mui/products.png" alt="" style="width: 427px; margin-bottom: 16px;" />
+
+### New package names
+
+The package names have changed in v5, which is a **breaking change**, you can find more details in the release notes.
+The npm organization name has moved from @material-ui to [@mui](https://www.npmjs.com/org/mui).
+
+### A new website & documentation
+
+We used this opportunity to tell a different story about the company and its products, positioning us in a broader space.
+We better explain and share what we do and who we are.
+We have created a custom brand identity, away from Material Design.
+For instance, you will find this custom color palette:
+
+<img loading="lazy" src="/static/blog/material-ui-is-now-mui/palette.png" alt="" style="width: 408px; margin-bottom: 16px;" />
+
+[Head to the new website](/).
+Note that these changes are **not** impacting the design of the components released under the `@mui/material` npm package.
+
+## The path ahead
+
+Most of this update has been rolled out, we will fix inconsistencies as we spot them.
```

Comment:
```suggestion
We have a lot of exciting stuff to come in the near future as we roll into this new chapter. We're grateful to have you together with us, supporting the library and making it grow bigger and bigger. If you spot any errors or inconsistencies, please warn us so we fix them.

Material Design will continue to be supported with the same attention to detail as before, nothing changes in regard to that. We hope to keep serving those who like the library for being loyal to MD while also leveraging the strong foundation built so far to offer new components, designs, and products. 

Hope you like it and happy building!
```

To address what @michaldudak said and also end the post with a closing/positive feel 😅